src/pybind/mgr/dashboard/plugins/pluggy.py

```diff
@@ -75,7 +75,7 @@ class _HookRelay(object):
     """
     def __init__(self):
         from collections import defaultdict
-        self._registry = defaultdict(list)
+        self._registry = defaultdict(list)  # type: ignore
```

Comment:
A `# type: DefaultDict[List]`does not work. Any ideas why?

Reply:
what do you get?

Reply:
Fixed with `# type: DefaultDict[str, list]`.